src/playwrightTestServer.ts

```diff
@@ -43,6 +44,12 @@ export class PlaywrightTestServer {
     this._disposeTestServer();
   }
 
+  async wsEndpoint() {
```

Comment:
It is unclear this starts the server. Let's make it more clear and specific: ensureStartedForTraceViewer(): and return trace viewer URL

Reply:
Addressed in 49d717f955696790e6d61abb7e91cf29ba63afc1